README.rst

```diff
@@ -1186,6 +1186,18 @@ Choose from the list of available rules:
   There should not be space before or after object ``T_OBJECT_OPERATOR``
   ``->``.
 
+* **operator_linebreak**
+
+  Binary operators must always be at the beginning or at the end of the
+  line.
```

Comment:
I think this misses "when multiline". Might be obvious to most, but I had a "what?" moment (the "ah hah" came soon after 😉)